### PR TITLE
[Storage] Support Point in Time Restore

### PR DESCRIPTION
**Pull request by blueww**
## Description

Design pass review in : 
preview change: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/466
GA update: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/704

The preview change has already released in preview module with PR to storage preview branch: https://github.com/Azure/azure-powershell/pull/11080

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by msJinLei — CHANGES_REQUESTED** [review]
Please fix the static analysis error

**Comment by blueww** [comment]
Add 2 static analysis exception:
1. `New-AzStorageBlobRangeToRestore` only create local object, so don't need ShouldProcess
2. "Enable-AzStorageBlobRestorePolicy" parameter `RestoreDays`, 

- this is aligned with other parameter of blob service, like RetentionDays pass review in https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/263, code in https://github.com/Azure/azure-powershell/blob/63b4ff6fff2191604b34b081c5ec853e29e474d5/src/Storage/Storage.Management/Blob/EnableAzureStorageBlobDeleteRetentionPolicy.cs#L85
- the name `RestoreDays` itself has pass review in Azure/azure-powershell-cmdlet-review-pr#466, and has been in preview release with PR #11080

File: src/Storage/Storage.Management/ChangeLog.md [hunk 1]
```diff
@@ -18,7 +18,14 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+* Support Point In Time Restore
```
Comment (msJinLei) [inline src/Storage/Storage.Management/ChangeLog.md:21]: please use past tense for "Support". thanks

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/Storage/Storage.Management/ChangeLog.md (hunk 1 above)
Comment (blueww) [reply src/Storage/Storage.Management/ChangeLog.md]: Fixed

**Review by msJinLei — APPROVED** [review]

**Pull request merged**